ReleaseHistory.md

```diff
@@ -17,6 +17,7 @@
 * DEP: Update Sarif.Sdk submodule from [120fae3 to 2d52c53](https://github.com/microsoft/sarif-sdk/compare/120fae3...2d52c53). Full [SARIF SDK Release History](https://github.com/microsoft/sarif-sdk/blob/2d52c53/src/ReleaseHistory.md).
 * BRK: Remove `--verbose` command-line option (in favor of `--level` and `--kind`). [#853](https://github.com/microsoft/binskim/pull/853)
 * BRK: Remove `--hashes` command-line option (in favor of `--insert Hashes`). [#853](https://github.com/microsoft/binskim/pull/853)
+* BUG: Fix `BA2004.EnableSecureSourceCodeHashing` where SHA1 always displayed if not SHA256 [#868](https://github.com/microsoft/binskim/pull/868)
```

Comment:
>where SHA1 always displayed if not SHA256 [](http://example.com/codeflow?start=50&length=41)

(don't change according my feedback on this)
this sounds perfect for a dev but I think we can make it longer and more clear like:

incorrectly always report that `SHA1` is used if the binary is not hashed with `SHA256`, while it could also be `MD5`.

Reply:
How about: Fixed issue where default hash is SHA1, when binary isn't compiled with SHA256.